<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinishShow</class>
 <widget class="QWidget" name="FinishShow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnPageUp">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>上一页</string>
       </property>
       <property name="icon">
        <iconset resource="png.qrc">
         <normaloff>:/png/png toolbar/Kombine_toolbar_007.png</normaloff>:/png/png toolbar/Kombine_toolbar_007.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPageDown">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>下一页</string>
       </property>
       <property name="icon">
        <iconset resource="png.qrc">
         <normaloff>:/png/png toolbar/Kombine_toolbar_025.png</normaloff>:/png/png toolbar/Kombine_toolbar_025.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFinish"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="png.qrc"/>
 </resources>
 <connections/>
</ui>
